app/_layout.tsx

```diff
@@ -43,6 +43,7 @@ const RootLayout = () => {
   const [fontsLoaded] = useFonts({
     /* eslint-disable unicorn/prefer-module,global-require */
     BelfastGrotesk_700Bold: require('@/assets/fonts/Belfast-Grotesk-Bold.otf'),
+    SourceCodePro_500Medium: require('@/assets/fonts/SourceCodePro-Medium.ttf'),
```

Comment:
FYI in enforcement backoffice, we used https://fonts.google.com/specimen/Space+Mono?preview.text=BA100OI